recipes/sre_yield/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "sre_yield" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/google/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: cd6fccf03dc5858a71f02c356fbb9546afd9de0de922898454b85f8955c69ef1
+
+build:
+  number: 0
```

Comment:
Can this be noarch?
```suggestion
  number: 0
  noarch: python
```

Reply:
Actually, I never tested the package on windows but, according to the log of the "Build recipe" stage for win_64, it seems that the package is dependant to a couple of system libraries. Thus, it shouldn't be noarch.